configuration/chairman/byron-shelley/configuration.yaml

```diff
@@ -8,6 +8,7 @@
 
 ByronGenesisFile: byron/genesis.json
 ShelleyGenesisFile: shelley/genesis.json
+AlonzoGenesisFile: alonzo/genesis.json
```

Comment:
New entry in the `configuration.yaml`

Reply:
If we include a new genesis file shouldn't we also include a hash in that file?

Reply:
You can but it's not mandatory.